recipes/conda-mirror/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "conda-mirror" %}
+{% set version = "0.6.5" %}
+{% set sha256 = "b609751148ffb5a1d36011d558673cb5eedf71327def05ea667c536004c1b0ca" %}
+
+{% set url_name = name.replace("-", "_") %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ url_name[0] }}/{{ url_name }}/{{ url_name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - conda-mirror = conda_mirror.conda_mirror:cli
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - requests
+    - pyyaml
+
+test:
+  imports:
+    - conda_mirror
+
+  commands:
+    - conda-mirror --help
+
+about:
+  home: https://github.com/ericdill/conda-mirror
+  license: BSD 3-Clause
+  license_family: BSD
+  # Can be uncommented after the upstream issue is addressed.
+  #
+  # https://github.com/maxpoint/conda-mirror/issues/36
+  #
+  # license_file: LICENSE
+  summary: 'mirror an upstream conda channel to a local directory'
+
+extra:
+  recipe-maintainers:
+    - ericdill
```

Comment:
Have added you as the maintainer, @ericdill. Please let me know if that is ok. If there are others that should be added, please let me know.